jsonnet/openshift-state-metrics.libsonnet

```
@@ -135,7 +140,7 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
       local containerVolumeMount = container.volumeMountsType;
       local podSelector = deployment.mixin.spec.template.spec.selectorType;
 
-      local podLabels = { 'k8s-app': 'openshift-state-metrics' };
+      local podLabels = $._config.commonLabels + { 'k8s-app': 'openshift-state-metrics' };
```

Now that we are using the recommended Kubernetes labels, we can remove `k8s-app`.
```suggestion
      local podLabels = $._config.commonLabels;
```

---

👍